code/modules/food_and_drinks/recipes/soup_mixtures.dm

```diff
@@ -85,6 +85,34 @@
 	if(!length(required_ingredients))
 		return
 
+	// If a food item is supposed to be made, remove relevant ingredients from the pot, then make the item 
+	if(resulting_food_path)
```

Comment:
```suggestion
	if(!isnull(resulting_food_path))
```
In the spirit of san who is perpetually telling me to do this, let's be specific.

Reply:
Well, I guess that's fine. If this is something we should be doing, perhaps it should be included in the code style guide.